internal/services/client/delivery.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"net/http"
+	"net/url"
+)
+
+//go:generate mockery  --name DeliveryMan
+type DeliveryMan interface {
+	Delivery(map[string]string) error
+}
+
+type LineMan struct {
```

Comment:
Лучше сделать структуру приватной для пакета, чтобы не было возможности обращаться к ней напрямую.
В проекте встречается такое, тоже можно поправить
```suggestion
type lineMan struct {
```